### DISCUSSION TITLE
Website button label

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#1568] [Q&A]
Is there a way to change the label of the website button to something like "Meeting script" rather than the start of the actual URL?

**Comment by gkovats** [comment]
What's your context or example URL? You mean in TSML Legacy or TSML UI?

**Comment by k8fish3r** [comment]
I've changed TSML Legacy — buttons on "meeting detail view" via Javascript. 

Definitely appreciate ideas on customizing buttons on "detail view" on TSML UI.

**Reply by district6wc** [reply]
Can I ask where you did that, what file needs modifying?

**Reply by k8fish3r** [reply]
We adjusted meeting detail buttons to qualify the "phone number" on them (rather than just call button.
Since swapped to single page / New UI which is great.

Hopeful there is a way to adjust the meeting detail pages sometime.

**Reply by gkovats** [reply]
@k8fish3r If you're still talking about TSML Legacy, we could possibly expose a WP filter where you'd have to add some `add_filter( 'tsml_view_meeting_phone', .... )` callback to change the details. But if you have 90 meetings, are you looking to change the button text for all to a single phrase, or make it contextual to the meeting's details?

Though, with TSML Legacy since it's all rendered from the backend, throwing on onload JavaScript on the page is _also_ an option. Just trying to understand your situation better.

**Reply by k8fish3r** [reply]
Apologies for my delay @gkovats . I managed to customise buttons in the legacy execution of 12 Step Meeting List. Since swapping to the React version I've not worked out a way to achieve the same amendments:
- E.g. expanding text on buttons (to include the phone number / email address).
- E.g. make web links within the meeting detail active / clickable URLS

If there's any route to control the behaviour of these types of elements it'd be really helpful.

Mindful the team get queries from loooads of teams supported by 12 SML component.

Kate F
